homeassistant/components/evohome/climate.py

```diff
@@ -149,7 +149,12 @@ def __init__(self, evo_broker, evo_device) -> None:
         """Initialize a Honeywell TCC Zone."""
         super().__init__(evo_broker, evo_device)
 
-        self._unique_id = evo_device.zoneId
+        if evo_device.modelType[:13] == "VisionProWifi":
```

Comment:
instead to generate a new array and copy the first 13 chars in, you can use:
`if evo_device.modelType.startswith("Vision...."): `

Reply:
Every day you learn something new!